feat: use same initial filter `owned` at TechDocsIndexPage as at CatalogPage (#12043)
backstage/backstage: .changeset/wicked-dragons-brush.md

---
'@backstage/plugin-techdocs': minor
---

> Use same initial filter `owned` at TechDocsIndexPage as at CatalogPage.

Reviewer: ```suggestion
Use same initial filter `owned` for the `TechDocsIndexPage` as for the `CatalogPage`.
```

> Use same initial filter `owned` for the `TechDocsIndexPage` as for the `CatalogPage`.

Reviewer 2: Maybe a note in here about "if you'd prefer to keep the default filter to `all`, do [x, y, z]`" could be useful?  It'd be nice to at least highlight that it's (more easily?) configurable now.
